### DISCUSSION TITLE
Electron+Vue3+Vite online & offline config for vue-cesium

### DISCUSSION THREAD

**Discussion by PetukhovArt** [zouyaoji/vue-cesium#589] [Q&A]
hi, zouyaoji ! 
It will be great if you can show the correct template project configuration for this.
First of all, interested in : vite configuration , import /build static files (core CesiumJS build)  when wee need working offline and it's dynamic import into Vue-Cesium from different .env variables.

Prod: offline mode
Dev : online mode
And in project we use different custom providers of layers , so i think :
1) with CesiumJS offline guide  [https://github.com/CesiumGS/cesium/tree/main/Documentation/OfflineGuide](url), we need to disable baseLayerPicker ?
2)  user can load his custom tile server to url-provider layer and use it , and we need to stop online requests from CesiumJS  in offline mode

Now, for hotfix, i paste CesiumJS build into my public folder and change the import path into vue-cesium. 

![image](https://github.com/user-attachments/assets/599d0cc4-ff5f-4f6a-82a8-5ff28481b895)

`
app.use(VueCesium, {
  locale: enUS,
  cesiumPath: `${CESIUM_PATH}/Cesium.js`, 
});
`

**Comment by zouyaoji** [comment]
Maybe you can take a look at these two projects.

https://github.com/zouyaoji/vue-cesium-electron-vite-starter

https://github.com/zouyaoji/vue-cesium-demo
